extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/sql/SqlMSQStatementResourcePostTest.java

```diff
@@ -139,30 +140,32 @@ public void testMSQSelectQueryTest() throws IOException
   @Test
   public void nonSupportedModes()
   {
-
     SqlStatementResourceTest.assertExceptionMessage(
         resource.doPost(new SqlQuery(
             "select * from foo",
             null,
             false,
             false,
             false,
-            ImmutableMap.of(),
```

Comment:
I think we should create a new sqlQuery object in the post request when you are mutating the map. WDYT?